src/DoubleTheDonation/Payment.php

```diff
@@ -52,6 +54,12 @@ public function addPaymentMeta( $payment_id, $payment_data ) {
 	 */
 	public function addDonationToDTD( $payment_id, $payment_data ) {
 
+        $donation = Donation::find($payment_id);
```

Comment:
@jonwaldstein something like this?

Reply:
@jonwaldstein resolved: https://github.com/impress-org/give-double-the-donation/pull/25/commits/107282fc6bc3fc6a86ed2305cad77c9c80d5d8bb